src/style/layouts/header.css

```diff
@@ -0,0 +1,76 @@
+.header-main {
+  position: absolute;
```

Comment:
Why do you use absolute here? Refrain to use absolute as least as you can. The position absolute just be used for the specific cases.